pkgs/servers/teleport/default.nix

```diff
@@ -46,6 +45,10 @@ buildGoPackage rec {
 
   buildTargets = [ "full" ];
 
+  preCheck = ''
+    export HOME=$(mktemp -d)
```

Comment:
Unit tests actually started running (?) after switching to `buildGoModule` and they seem to expect a proper `$HOME` to be set.

Reply:
Yeah, it enabled a default checkPhase but thats good.